main.go

```diff
@@ -144,6 +144,7 @@ func main() {
 		log.Error("Failed to create SubnetManager: ", err)
 		os.Exit(1)
 	}
+	log.Info(fmt.Sprintf("Created subnet manager: %+v", sm))
```

Comment:
I did initially, I caught it and squashed in to the previous commit.  Looks ok on the remote branch now.